package.json

```diff
@@ -12,7 +12,7 @@
     "store"
   ],
   "scripts": {
-    "test": "node --experimental-vm-modules node_modules/.bin/jest --coverage && eslint . && check-dts && size-limit && yaspeller *.md"
+    "test": "node --experimental-vm-modules node_modules/.bin/jest && eslint . && check-dts && size-limit && yaspeller *.md"
```

Comment:
We can’t remove coverage. I use it to be sure that I tested everything.

What part of the code you can’t cover?

Reply:
sorry, leaked from local testing